css/index.css

```
@@ -1,693 +1,1073 @@
+
+.visually-hidden {
   position: absolute;
-  top: 0;
-  left: 0;
-  z-index: -1;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.4.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)